masonry/src/theme.rs

```diff
@@ -92,6 +92,27 @@ pub fn default_property_set() -> DefaultProperties {
         Gradient::new_linear(0.0).with_stops([DISABLED_BUTTON_LIGHT, DISABLED_BUTTON_DARK]),
     )));
 
+    let checkbox_gradient =
+        Gradient::new_linear(0.0).with_stops([BACKGROUND_DARK, BACKGROUND_LIGHT]);
+    properties.insert::<Checkbox, _>(BorderWidth { width: 1.0 });
+    properties.insert::<Checkbox, _>(BorderColor { color: BORDER_DARK });
+    properties.insert::<Checkbox, _>(HoveredBorderColor(BorderColor {
+        color: BORDER_LIGHT,
+    }));
+    properties.insert::<Checkbox, _>(CornerRadius { radius: 2. });
+    properties.insert::<Checkbox, _>(Background::Gradient(checkbox_gradient.clone()));
+    properties.insert::<Checkbox, _>(ActiveBackground(Background::Gradient(
+        checkbox_gradient.clone(),
+    )));
+    properties.insert::<Checkbox, _>(DisabledBackground(Background::Gradient(
+        checkbox_gradient.clone(),
+    )));
+    properties.insert::<Checkbox, _>(CheckmarkWidth { width: 2.0 });
+    properties.insert::<Checkbox, _>(CheckmarkColor { color: TEXT_COLOR });
+    properties.insert::<Checkbox, _>(DisabledCheckmarkColor(CheckmarkColor {
+        color: DISABLED_TEXT_COLOR,
+    }));
```

Comment:
Always having to repeat `properties.insert::<Checkbox, _>` here doesn't look nice. I think it would be neat if we could avoid that somehow.

Reply:
I think some amount of copy-pasting here is fine for the time being. It's a fairly small file anyway, and logically very simple.